erfa/ufunc.c.templ

```diff
@@ -734,7 +734,12 @@ PyMODINIT_FUNC PyInit_ufunc(void)
            as these do not get copied */ #}
     {%- for func in funcs %}
     {%- if not func.user_dtype %}
-    static char types_{{ func.pyname }}[{{ func.args_by_inout('in|inout|out|ret|stat')|count }}] = { {{ func.args_by_inout('in|inout|out|ret|stat')|map(attribute='npy_type')|join(', ') }} };
```

Comment:
This previous line didn't take into account that for the ufunc, each 'inout' element is split in an 'in' and 'out' element. As a result, for `rz`, it had 
```
static char types_rz[2] = {NPY_DOUBLE, NPY_DOUBLE};
```
i.e., it missed one.  Apparently, on MacOS only the extra one became bool (some alignment perhaps?)